### PR TITLE
[NO-TICKET] Folder with localized manifests

### PR DESCRIPTION
**Pull request by laubengaier**
# Description
Adding folder with localized manifests

# Checklist
- [x] I wrote/updated tests for new/changed code.
- [x] I ran `yarn sdk buildSdk` and `yarn bridge buildBridge` without issues.
- [x] I ran `yarn sample prettify` and `yarn sample build` without issues.

### CODE REVIEW

**Review by corycaywood — COMMENTED** [review]
@laubengaier This format for the manifest works, but I think it would be better to use the placeholder format. That way we don't have to duplicate content across the files:

```

// manifest.json:
{
    "customMetaData": {
        "exampleKey": "__MANIFEST_exampleText__",
    }
}
 
// _locales/default/manifest.json:
{
  "exampleText": "Default text."
}
 
// _locales/ja/manifest.json:
{
  "exampleText": "Japanese text."
}
```

File: js-miniapp-sample/public/_locale/ja/manifest.json [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    {
+      "name": "rakuten.miniapp.user.PROFILE_PHOTO",
+      "reason": "Describe your reason here (optional)."
+    }
+  ],
+
+  "optPermissions": [
+    {
+      "name": "rakuten.miniapp.user.CONTACT_LIST",
+      "reason": "Describe your reason here (optional)."
```
Comment (corycaywood) [inline js-miniapp-sample/public/_locale/ja/manifest.json:30]: Maybe we should add a translation for the `reason` too?
I google translated this 😆 : `ここに理由を説明してください（オプション）。`

**Review by rleojoseph — CHANGES_REQUESTED** [review]
@laubengaier 
`manifest.json` from the main folder should contain the key for these localized texts
```

  {
      "name": "rakuten.miniapp.user.USER_NAME",
      "reason": "__MANIFEST_userNameReason__"
  },
  {
      "name": "rakuten.miniapp.user.POINTS",
      "reason": "__MANIFEST_pointsReason__"
  }
```

And the `manifest.json` inside _locale folders will contain only the value of those keys, like

_locale/default/manifest.json

```
{
  "userNameReason": "Default text.",
  "pointsReason: "Default text"
}
```

_locale/ja/manifest.json

```
{
  "userNameReason": "JP Default text.",
  "pointsReason: "JP Default text"
}
```

File: js-miniapp-sample/public/_locale/ja/manifest.json (hunk 1 above)
Comment (laubengaier) [reply js-miniapp-sample/public/_locale/ja/manifest.json]: :D yeah I think that works (anyways demo purpose)

**Review by rleojoseph — APPROVED** [review]
Looks good to me :+1:

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Pull request merged**